# HG changeset patch
# User alta88@fixall.com
# Date 1701202127 -3600
# Parent  aebf62b8e3b4127e962254b76cff6a1110944cec
Bug 1855611 - message list / thread pane can't be resized smaller.

One row should be visible, minimally.
Also, fix some min heights on elements which should not shrink when sizing to a small height.

diff --git a/mail/themes/shared/mail/about3Pane.css b/mail/themes/shared/mail/about3Pane.css
--- a/mail/themes/shared/mail/about3Pane.css
+++ b/mail/themes/shared/mail/about3Pane.css
@@ -126,17 +126,17 @@ body {
   grid-area: folderPaneSplitter;
 }
 
 #threadPane {
   container-name: threadPane;
   container-type: inline-size;
   grid-area: threads;
   box-sizing: border-box;
-  min-height: 200px;
+  min-height: 100px;
   min-width: 300px;
   display: flex;
   flex-direction: column;
   overflow: hidden;
 }
 
 #messagePaneSplitter {
   grid-area: messagePaneSplitter;
diff --git a/mail/themes/shared/mail/messageHeader.css b/mail/themes/shared/mail/messageHeader.css
--- a/mail/themes/shared/mail/messageHeader.css
+++ b/mail/themes/shared/mail/messageHeader.css
@@ -287,16 +287,17 @@ mail-tagfield[collapsed="true"] {
 
 splitter[orient="vertical"] {
   appearance: none;
   background-color: transparent;
   border-width: 0;
   border-top: 1px solid var(--splitter-color);
   /* splitter grip area */
   height: 5px;
+  min-height: 5px;
   /* make only the splitter border visible */
   margin-bottom: -5px;
   /* because of the negative margin needed to make the splitter visible */
   position: relative;
   z-index: 10;
 }
 
 @media (prefers-reduced-motion: no-preference) {
@@ -312,16 +313,17 @@ splitter[orient="vertical"]:focus {
 
 splitter[state="collapsed"]:not(:hover, :focus) {
   border-top-width: 0;
 }
 
 /* ::::: attachment view ::::: */
 
 #attachmentBar {
+  min-height: inherit;
   color: var(--layout-color-1);
   background-color: var(--layout-background-1);
   padding: 3px 0;
   overflow: hidden;
 }
 
 /* Collapsed list */
 #attachmentView[location="top"] #attachmentBar[tooltiptext^="Show"] {
